torch/testing/_internal/common_nn.py

```diff
@@ -3410,7 +3752,8 @@ def padding3d_circular(input, pad):
         module_name='CTCLoss',
         desc='2d_int_target',
         constructor_args=(0,),  # blank=0
-        extra_args=([50, 50, 50], [30, 25, 20]),  # input_lengths, target_lengths
+        cpp_constructor_args='torch::nn::CTCLossOptions().blank(0)',
+        extra_args=(torch.tensor([50, 50, 50]), torch.tensor([30, 25, 20])),  # input_lengths, target_lengths
```

Comment:
This makes it so that we aren't testing that CTCLoss can take non-Tensor input_lengths. This could be a problem if someone unintentionally changes the API here.

I'm still trying to figure out how `extra_args` gets used by the cpp parity test so I have no suggestions here yet.

Reply:
We always serialize `extra_args` as a list of tensors and load them into the cpp parity test, and changing it to also allow serializing lists of ints might be too ad-hoc. Maybe as a workaround, we can make a copy of this module test and use `extra_args=(torch.tensor([50, 50, 50]), torch.tensor([30, 25, 20]))`, and for the original one we put `test_cpp_api_parity=False`?

Reply:
Yeah, I think it would be good to add a copy of the module test as a workaround to make sure that works

Reply:
Added :D